skops/hub_utils/_hf_hub.py

```diff
@@ -291,6 +318,9 @@ def init(
         :class:`numpy.ndarray`. If ``task`` is ``"text-classification"`` or
         ``"text-regression"``, the data needs to be a ``list`` of strings.
 
+    model_format: str
+        The format used to persist the model. Can be ``"auto"``, ``"skops"``
```

Comment:
Maybe it's just me, but when I read this docstring, it could also mean "the model format that is being used [by this function] to persist the model", when what it really means is "the model format that was used to persist the model [by the user]". The former interpretation could lead the user to believe that `init` will create a new model file using that format. E.g. when I have a pickle file and I use `model_format="skops"`, then `init` will create a skops file from the pickle file for me.

Something like "The format used by the persisted model" could help to disambiguate the two meanings. WDYT?

The same ambiguity can be found in the docstring of `_create_config` but there it's less important, as it's internal.